src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class RightTriangle extends AbstractFigure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line above the class declaration.